ui/src/components/forms/FormManager.js

```diff
@@ -150,20 +147,26 @@ const FormManager = (props) => {
         <Box borderRadius="lg">
           <HStack>
             <Heading mb="2" as="h2" size="lg" align="left">
-              Existing Forms 
+              Existing Forms
             </Heading>
           </HStack>
-          <List spacing="2">
+          <SimpleGrid spacing="2" columns={[1, 2]}>
             {formList.map((formItem) => (
-              <ListItem key={formItem.id}>
+              <Box key={formItem.id}>
                 <Box p="3" borderRadius="lg" borderWidth="1px">
                   <Text mb="3">
                     <Badge colorScheme={formItem.isPublished ? 'green' : 'red'}>
                       {formItem.isPublished ? 'LIVE' : 'PRIVATE'}
                     </Badge>{' '}
+                    <Badge>
+                      {formItem.formType ? formItem.formType : 'internal'}
```

Comment:
when clicking external form 'view form' in admin page it doesn't work
but it does work on my profile page. wondering if this is intended behaviour